### PR TITLE
set web-ci-pipeline.yml only triggered by a few subfolders

### PR DESCRIPTION
**Pull request by jchen351**
### Description
Isolating Web-CI pipeline from rest of the pipeline. In this case, we update made to the WebCi only, the rest of pipelines for other EP will not be triggerred

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml [hunk 1]
```diff
@@ -1,3 +1,49 @@
+trigger:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    include:
+    - 'js/web'
```
Comment (fs-eire) [inline tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml:8]: everything in js/ except js/react_native should trigger web CI

**Comment by mszhanyi** [comment]
Personally， it'd be better not to complicate includes and excludes too much.
I'm afraid that if there's any changes in the future no one can change the rules correctly.

**Comment by mszhanyi** [comment]
https://github.com/Microsoft/azure-pipelines-agent/issues/1580
https://stackoverflow.com/questions/59099507/anchors-are-not-currently-supporting-while-running-azure-devops-yml-file

File: tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml [hunk 2]
```diff
@@ -16,34 +15,17 @@ trigger:
...
-    - '.github'
-
-    exclude:
+    - 'js/react_native'
     - 'onnxruntime/core/providers/tensorrt'
     - 'onnxruntime/core/providers/cuda'
     - 'onnxruntime/core/providers/rocm'
     - '!onnxruntime/core/providers/rocm/rocm_provider_factory_creator.h'
     - '!onnxruntime/core/providers/cuda/cuda_provider_factory_creator.h'
     - '!onnxruntime/core/providers/tensorrt/tensorrt_provider_factory_creator.h'
```
Comment (fs-eire) [inline tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml:24]: just curious: why these 3 files are needed?

**Pull request closed without merging**